core/templates/pages/exploration-player-page/templates/information-card-modal.directive.html

```diff
@@ -108,4 +108,21 @@ <h2 ng-style="titleWrapper()" class="oppia-info-card-title"><[explorationTitle]>
     cursor: default;
     margin-right: 5px;
   }
+  .oppia-info-card-content p {
+    line-height: 1.5;
+    padding-bottom: 8px;
+  }
+  .oppia-info-card-bottom-row {
+    line-height: 1.5;
+  }
+  .flex-40 {
+    padding-bottom: 2%;
+  }
+  .oppia-info-card-content .card-metrics li .far {
+    margin-right: 0;
```

Comment:
Hm, if this is the only style property, can we use a bootstrap class in the relevant element ([example](https://getbootstrap.com/docs/4.4/utilities/spacing/))?

Reply:
yeah sure, We can include " .mr-0 " .I'll do that.